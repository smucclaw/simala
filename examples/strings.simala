foobar = "foo" ++ "bar";
number = 3 + 5;

#eval foobar;
#eval explode(foobar);
#eval number
