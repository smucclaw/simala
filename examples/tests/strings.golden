"foobar"
["f","o","o","b","a","r"]
8
